[Bug] Bug report

### Library used

@paypal/react-paypal-js


<!-- paypal-js, react-paypal-js, direct script integration -->

### 🐞 Describe the Bug

onur@onur$ npm install @paypal/react-paypal-js
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: valet-booking@0.1.0
npm ERR! Found: react@19.0.0-rc-66855b96-20241106
npm ERR! node_modules/react
npm ERR!   react@"19.0.0-rc-66855b96-20241106" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0 || ^17.0.0 || ^18.0.0" from @paypal/react-paypal-js@8.7.0
npm ERR! node_modules/@paypal/react-paypal-js
npm ERR!   @paypal/react-paypal-js@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /home/onur/.npm/_logs/2024-11-24T04_09_13_719Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/onur/.npm/_logs/2024-11-24T04_09_13_719Z-debug-0.log

### 🔬 Minimal Reproduction

npm install @paypal/react-paypal-js inside next js 15 project

### 😕 Actual Behavior

A clear and concise description of what is happening. Please include console logs during the time of the issue, especially error messages.

### 🤔 Expected Behavior

npm install paypal react and then implement it

### 🌍 Environment

-

### ➕ Additional Context

One Stripe react component library also isnt also compatible with react 19 via npm but i have used a workaround with stripe so i dont need to downgrade my project

--- a/.changeset/grumpy-tigers-build.md
+++ b/.changeset/grumpy-tigers-build.md
@@ -1,0 +1,5 @@
+---
+"@paypal/react-paypal-js": patch
+---
+
+minor fixes to release and adding react 19 as peer dependency

--- a/.github/workflows/playwright.yml
+++ b/.github/workflows/playwright.yml
@@ -39,7 +39,7 @@ jobs:
             - name: ▶️ Run Playwright tests
               run: npm run test:e2e
 
-            - uses: actions/upload-artifact@v3
+            - uses: actions/upload-artifact@v4
               if: always()
               with:
                   name: playwright-report

--- a/packages/react-paypal-js/package.json
+++ b/packages/react-paypal-js/package.json
@@ -103,8 +103,8 @@
         "typescript": "^4.7.2"
     },
     "peerDependencies": {
-        "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
-        "react-dom": "^16.8.0 || ^17.0.0 || ^18.0.0"
+        "react": "^16.8.0 || ^17 || ^18 || ^19",
+        "react-dom": "^16.8.0 || ^17 || ^18 || ^19"
     },
     "jest": {
         "transformIgnorePatterns": [

--- a/packages/react-paypal-js/tsconfig.declarations.json
+++ b/packages/react-paypal-js/tsconfig.declarations.json
@@ -1,5 +1,5 @@
 {
     "extends": "./tsconfig.json",
     "include": ["src"],
-    "exclude": ["src/stories"]
+    "exclude": ["src/stories", "src/**/*.test.ts", "src/**/*.test.tsx"]
 }

--- a/scripts/check-node-version.js
+++ b/scripts/check-node-version.js
@@ -3,7 +3,7 @@ const { join } = require("path");
 const semver = require("semver");
 
 const expectedNodeVersion = readFileSync(
-    join(__dirname, "../", ".nvmrc"),
+    join(__dirname, "../../", ".nvmrc"),
     "utf-8",
 );
 const isValidNodeVersion = semver.satisfies(

